### PR TITLE
fix wiki build errors/links on tricopter page

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: copter/source/docs/tricopter.rst [hunk 1]
```diff
@@ -35,22 +35,22 @@ Copter 3.4.x
 
 - :ref:`MOT_YAW_SV_ANGLE <MOT_YAW_SV_ANGLE>` : tail servo's maximum lean angle in degrees.  This allows for the rear motor's thrust to be adjusted appropriately depending upon the lean angle.  The default is 30 degrees.  "0" would mean the tail servo can only point directly up (which would not allow the vehicle to fly), "90" means the tail servo can point horizontally.
 
-The channel used for the tail servo can be changed from it's default (channel 7) by setting the appropriate RCX_FUNCTION to 39.  For example the :ref:`Pixracer <common-pixracer-overview>` only has 6 output channels so the tail servo can be moved to output channel 5 by setting :ref:`RC5_FUNCTION <RC5_FUNCTION>` to 39.
+The channel used for the tail servo can be changed from it's default (channel 7) by setting the appropriate RCX_FUNCTION to 39.  For example the :ref:`Pixracer <common-pixracer-overview>` only has 6 output channels so the tail servo can be moved to output channel 5 by setting :ref:`SERVO5_FUNCTION <SERVO5_FUNCTION>` to 39.
```
Comment (rmackay9) [inline copter/source/docs/tricopter.rst]: I think this RCX_FUNCTION should probably be changed to be SERVOx_FUNCTION

**Comment by rmackay9** [comment]
Merged, thanks!  There's an issue earlier in the paragraph but the change is good and fixes the build error which was the purpose.

**Comment by Hwurzburg** [comment]
I  will fix it...you mean it should be SERVOx _FUNCTION, right?

**Comment by rmackay9** [comment]
@Hwurzburg, yes, that's right, txs

**Pull request merged**